central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -49,11 +57,76 @@ func (s *fullStoreImpl) GetProcessListeningOnPort(
 		"ProcessListeningOnPortStorage",
 	)
 
+	allowed, err := pgutils.Retry2(func() (bool, error) {
+		return s.checkAccess(ctx, deploymentID)
+	})
+
+	if err != nil {
+		log.Info("Returning nil, err")
```

Comment:
Think about cleaning the debug logs before merging.

Reply:
Oops. Done.